src/adapter/src/client.rs

```diff
@@ -53,6 +56,23 @@ impl Handle {
     }
 }
 
+#[derive(Debug, Clone)]
+struct Metrics {
+    pub commands: IntCounterVec,
+}
+
+impl Metrics {
+    pub fn register_into(registry: &MetricsRegistry) -> Metrics {
+        Metrics {
+            commands: registry.register(metric!(
+                name: "mz_adapter_commands",
+                help: "The total number of adapter commands issued  of the given type since process start.",
```

Comment:
Not a huge deal but it looks like there might be two spaces in between `issued` and `of`.

Reply:
Fixed.